demo/midgard-sdk/src/tx-builder/user-events/deposit/types.ts

```diff
@@ -1,22 +1,39 @@
-import { Address, PolicyId, Data, UTxO } from "@lucid-evolution/lucid";
-import { POSIXTimeSchema } from "@/tx-builder/common.js";
+import { Data } from "@lucid-evolution/lucid";
+import { OutputReferenceSchema, POSIXTimeSchema } from "@/tx-builder/common.js";
 
-export const DepositDatumSchema = Data.Object({
-  data: Data.Object({
-    startTime: POSIXTimeSchema,
-    endTime: POSIXTimeSchema,
-  }),
+export const DepositEventSchema = Data.Object({
+    id: OutputReferenceSchema,
+    info: Data.Object({
+      l2_address: Data.Bytes(),
+      l2_datum: Data.Nullable(Data.Bytes())
+    })
```

Comment:
We now should be able to use `DepositInfoSchema` here:
```suggestion
    info: DepositInfoSchema,
```